content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -17,13 +17,154 @@ aliases:
+
+  `format=car` &rarr; `Accept: application/vnd.ipld.car`
+
+- `depthType` - _Optional_. Used to specify the depth of the DAG to return in the response.
+
+    - `depthType=full` - Returns the entire DAG from the termination of the `{cid}[/path]` specifier, as well as all blocks from the `cid` to the `path` terminus where a `path` is provided. This is the default behavior when no `depthType` is provided.
+
+    - `depthType=shallow` - Returns only the content at the termination of the `{cid}[/path]` specifier, as well as all blocks from the `cid` to the `path` terminus where a `path` is provided. If the content is found to be UnixFS data, the entire UnixFS entity will be included. i.e. if `{cid}[/path]` terminates at a sharded UnixFS file, or a sharded UnixFS directory, the blocks required to reconsititute the entire file, or directory will be included. If the termination is a UnixFS sharded directory, only the full directory will be included, not the full DAG of the directory's contents.
+
+#### Response
+
+#### Status Codes
+
+- `200` - OK
+
+- `400` - Bad Request
+    - No acceptable content type provided in the `Accept` header
+    - Requested a non-supported format via the `format` query parameter
+    - Neither providing a valid `Accept` header or `format` query parameter
+    - No extension given in `filename` query parameter
+    - Used a non-supported extension in the `filename` query parameter
+
+- `404` - No candidates for the given CID were found
+
+- `500` - Internal Server Error
+    - The requested CID path parameter could not be parsed
+    - An internal retrieval ID failed to generate
+    - The internal blockstore file failed to write
+
+- `504` - Timeout occured while retrieving the given CID
+
+##### Headers
+
+- `Accept-Ranges` - Returns with `none` if the block order in the CAR stream is not deterministic
+
+- `Cache-Control` - Returns with `public, max-age=29030400, immutable`
+
+- `Content-Disposition` - Returns as an attachment, using the given `filename` query parameter if provided, or if no `filename` query parameter is provided, uses the requested CID with a `.car` extension.
+
+  Example: `bafy...foo.car`
+
+- `Content-Type` - Returns with `application/vnd.ipld.car; version=1`
+
+- `Etag` - Returns with the requested CID with the format as a suffix.
+
+  Example: `bafy...foo.car`
+
+- `X-Content-Type-Options` - Returns with `nosniff` to indicate that the `Content-Type` should be followed and not to be changed. This is a security feature, ensures that non-executable binary response types are not used in `<script>` and `<style>` HTML tags.
+
+- `X-Ipfs-Path` - Returns the original, requested content path before any path resolution and traversal is performed.
+
+  Example:  `/ipfs/bafy...foo`
+
+- `X-Trace-ID` - Returns the given `X-Request-Id` header value if provided, otherwise returns an ID that uniquely identifies the retrieval request.
+
+### Lassie Car files
```

Comment:
```suggestion
### Lassie's CAR format
```

If we can avoid "CAR **file**" that would be good, it's a "transport" that often ends up as a file, but shouldn't be conceived only as a file. As per #1833, you don't even need to treat it as a file to use it. I know it's hard to avoid "file" in many cases though.

Reply:
I think also this section is too much, they could just be pointed to https://github.com/filecoin-project/lassie/blob/main/docs/CAR.md for the gory details. From the user perspective they get a CAR, they can unpack files from that or pass it on to somewhere else. A lot of the details here are useful if you're building on top of lassie, but I imagine most readers of these docs just want to run a CLI to fetch and unpack the content they stored.